### PR TITLE
Updates selection controls gallery demo

### PR DESCRIPTION
**Pull request by vlidholt**

### CODE REVIEW

File: examples/material_gallery/lib/demo/selection_controls_demo.dart [hunk 1]
```diff
@@ -4,74 +4,241 @@
...
+  "Radio buttons allow the user to select one option from a set. Use radio "
+  "buttons for exclusive selection if you think that the user needs to see "
+  "all available options side-by-side.";
+
+const String _radioCode =
+"""// Member variable holding value.
+int radioValue = 0;
+
+// Method setting value
+void setRadioValue(int value) {
```
Comment (abarth) [inline examples/material_gallery/lib/demo/selection_controls_demo.dart]: We'd usually call this handleRadioValueChanged

File: examples/material_gallery/lib/demo/selection_controls_demo.dart [hunk 2]
```diff
@@ -4,74 +4,241 @@
...
-      _radioValue = value;
-    });
-  }
+  bool checkboxValueA = true;
+  bool checkboxValueB = false;
+  int radioValue = 0;
+  bool switchValue = false;
 
-  void _setSwitchValue(bool value) {
+  void _setRadioValue(int value) {
```
Comment (abarth) [inline examples/material_gallery/lib/demo/selection_controls_demo.dart]: _handleRadioValueChanged :)

File: examples/material_gallery/lib/demo/selection_controls_demo.dart [hunk 3]
```diff
@@ -4,74 +4,241 @@
...
+  onChanged: (bool value) {
+    setState(() {
+      checkboxValue = value;
+    }
+  );
+})
+
+// Create a disabled checkbox.
+// Checkboxes are disabled when onChanged isn't
+// specified or null
```
Comment (HansMuller) [inline examples/material_gallery/lib/demo/selection_controls_demo.dart]: .

File: examples/material_gallery/lib/demo/selection_controls_demo.dart [hunk 4]
```diff
@@ -4,74 +4,241 @@
...
 import 'package:flutter/material.dart';
 
+import '../gallery/demo.dart';
+
+const String _checkboxText =
+  "# Checkboxes\n"
+  "Checkboxes allow the user to select multiple options from a set.";
+
+const String _checkboxCode =
+"""// Member variable holding value.
```
Comment (HansMuller) [inline examples/material_gallery/lib/demo/selection_controls_demo.dart]: holding the checkbox's value

File: examples/material_gallery/lib/demo/selection_controls_demo.dart [hunk 5]
```diff
@@ -4,74 +4,241 @@
...
+      alignment: new FractionalOffset(0.5, 0.4),
+      child: new Row(
+        mainAxisAlignment: MainAxisAlignment.collapse,
+        children: <Widget>[
+          new Switch(value: switchValue, onChanged: (bool value) {
+            setState(() {
+              switchValue = value;
+            });
+          }),
+          new Switch(value: true),
```
Comment (HansMuller) [inline examples/material_gallery/lib/demo/selection_controls_demo.dart]: The original code, here and elsewhere, included a `// Disabled` comment for these cases. Seems reasonable to keep that.

**Comment by HansMuller** [comment]
LGTM

**Pull request merged**